core/types/istanbul.go

```diff
@@ -50,10 +51,16 @@ type IstanbulExtra struct {
 	Seal []byte
 	// Bitmap is a bitmap having an active bit for each validator that signed this block
 	Bitmap *big.Int
-	// CommittedSeal is an aggregated BLS signature resulting from signatures by each validator that signed this block
+	// CommittedSeal is an aggregated BLS signature resulting from signatures by
+	// each validator that signed this block
 	CommittedSeal []byte
 	// EpochData is a SNARK-friendly encoding of the validator set diff (WIP)
 	EpochData []byte
+	// ParentBitmap is a bitmap having an active bit for each validator that
+	// signed on the previous block
+	ParentBitmap *big.Int
+	// ParentCommit is the aggregated BLS signature resulting from commits of the validators which signed on the previous block
+	ParentCommit []byte
```

Comment:
Wdyt of `ParentSeal` here? Would be more consistent with `CommittedSeal`

Reply:
Had originally called it "ParentCommits" because of: https://github.com/celo-org/celo-blockchain/pull/538#discussion_r337616746. Either name works for me.